sdk/src/main/java/dev/cdevents/CDEvents.java

```diff
@@ -4,10 +4,7 @@
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.node.ObjectNode;
-import com.networknt.schema.JsonSchema;
-import com.networknt.schema.JsonSchemaFactory;
-import com.networknt.schema.SpecVersion;
-import com.networknt.schema.ValidationMessage;
```

Comment:
Please revert back to explicit imports. Do not use star imports

Reply:
reverted and applied to individual imports